spec/api/api.yaml

```diff
@@ -185,6 +197,43 @@ paths:
           $ref: '#/components/responses/Error'
         '500':
           $ref: '#/components/responses/Error'
+  /getDocument:
```

Comment:
Gosh I'm sorry to see this go in.  I predict that it will come back to haunt us.

... we should discuss getting rid of the whole server-side snapshot/compare facility...

Reply:
This is the simplest solution compared to the others.